documentation/library_guide/macros.rst

```diff
@@ -87,12 +87,12 @@ Macro                              Description
                                    If all parallel backends are disabled by setting respective macros to 0, algorithms
                                    with parallel policies are executed sequentially by the calling thread.
 ---------------------------------- ------------------------------
-``ONEDPL_USE_DPCPP_BACKEND``       This macro enables the use of the SYCL* policies.
+``ONEDPL_USE_DPCPP_BACKEND``       This macro enables the use of the device policies.
```

Comment:
In this line, let's use "device execution policies"; this is the first mentioning in the document, and it makes sense to accent the "full term".